elide-contrib/elide-test-helpers/src/main/java/com/yahoo/elide/contrib/testhelpers/jsonapi/elements/Data.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2019, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+
+package com.yahoo.elide.contrib.testhelpers.jsonapi.elements;
+
+import com.google.gson.Gson;
+
+import java.util.LinkedHashMap;
+
+/**
+ * The type Data.
+ */
+public class Data extends LinkedHashMap<String, Object> {
+    static private final Gson GSON_INSTANCE = new Gson();
```

Comment:
Is there a reason we switched from jackson everywhere else to gson here?

Reply:
This code originally used gson and I saw the swagger project was using gson too: 

https://github.com/yahoo/elide/blob/7fe0a5730ffa520c16ec4a62b5ee04f2bc42142a/elide-contrib/elide-swagger/pom.xml#L67 

I can try to see if jackson works.